flink-runtime/src/test/java/org/apache/flink/runtime/metrics/dump/MetricQueryServiceTest.java

```diff
@@ -106,14 +104,14 @@ public void testCreateDump() throws Exception {
         MetricDumpSerialization.MetricSerializationResult emptyDump =
                 queryService.queryMetrics(TIMEOUT).get();
 
-        assertEquals(0, emptyDump.serializedCounters.length);
-        assertEquals(0, emptyDump.serializedGauges.length);
-        assertEquals(0, emptyDump.serializedHistograms.length);
-        assertEquals(0, emptyDump.serializedMeters.length);
+        assertThat(emptyDump.serializedCounters.length).isZero();
```

Comment:
can we use `assertThat(emptyDump.serializedCounters).isEmpty();` here? which is similar to line 94.

Reply:
fixed